usaspending_api/search/v2/views/spending_by_category.py

```diff
@@ -74,6 +74,14 @@ def post(self, request: Request) -> Response:
         models = [
             {"name": "category", "key": "category", "type": "enum", "enum_values": categories, "optional": False},
             {"name": "subawards", "key": "subawards", "type": "boolean", "default": False, "optional": True},
+            {
+                "name": "spending_level",
+                "key": "spending_level",
+                "type": "enum",
+                "enum_values": ["awards", "transactions", "subawards"],
```

Comment:
Would recommend using the same list of enum values that you used below:
`"enum_values": [level.value for level in SpendingLevel],`